app/oauth2/access-token-request.js

```diff
@@ -1,5 +1,8 @@
+const { Logger } = require('@hmcts/nodejs-logging');
```

Comment:
No need for `{ }` if using `require`. You just need `const Logger = require(...)`